cmd/src/campaign_progress_printer.go

```diff
@@ -134,13 +134,30 @@ func (p *campaignProgressPrinter) PrintStatuses(statuses []*campaigns.TaskStatus
 	}
 
 	for _, ts := range newlyCompleted {
+		p.progress.WriteLine(output.Linef("", output.StylePending, "%s", ts.RepoName))
+		lines, err := diffLines(ts)
+		if err != nil {
+			p.progress.Verbosef("%-*s failed to display status: %s", p.maxRepoName, ts.RepoName, err)
+			continue
+		}
+		for _, line := range lines {
+			p.progress.Verbosef("\t%s", line)
+		}
+
```

Comment:
Do we want to gate this on whether verbose mode is enabled, so we don't have to calculate the diffstat otherwise?

Reply:
Done!